sumologic/provider.go

```diff
@@ -18,14 +18,19 @@ func Provider() terraform.ResourceProvider {
 		Schema: map[string]*schema.Schema{
 			"access_id": {
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
```

Comment:
I am not sure if  we want to make them optional. This is still our primary way to authenticate. We can set to them to some dummy string for our use case.

Reply:
Ok, changing them back to `Required`, and changing the checking logic accordingly